libi2pd/RouterInfo.cpp

```diff
@@ -553,6 +558,20 @@ namespace data
 		return bufbe64toh (buf + size) > m_Timestamp;
 	}
 
+	const uint8_t * RouterInfo::LoadBuffer(const uint8_t *buf, size_t len) //TODO: error handling
```

Comment:
what does LoadBuffer do?

Reply:
Basically, it copies buf to m_Buffer and calls RouterInfo::LoadBuffer without checking signature, like when we are loading from file.